roverrobotics_driver/include/roverrobotics_ros_driver.hpp

```diff
@@ -16,61 +20,63 @@
 
 namespace RoverRobotics {
 
-class ROSWrapper {
+class RobotDriver {
  private:
-  int motors_id_[4];
-  //
   std::unique_ptr<BaseProtocolObject> robot_;
   // Pub Sub
   ros::Subscriber speed_command_subscriber_;  // listen to cmd_vel inputs
+  ros::Subscriber mode_trigger_subscriber_;   // listen to change mode inputs
   ros::Subscriber trim_command_subscriber_;   // listen to trim value broadcast
   ros::Subscriber estop_trigger_subscriber_;  // listen to estop trigger inputs
   ros::Subscriber estop_reset_subscriber_;    // listen to estop reset inputs
 
-  ros::Subscriber robot_info_subscriber;  // listen to robot_info request
-  ros::Publisher robot_info_publisher;    // publish robot_unique info
+  ros::Subscriber robot_info_subscriber_;  // listen to robot_info request
+  ros::Publisher robot_info_publisher_;    // publish robot_unique info
 
   ros::Publisher robot_status_publisher_;  // publish robot state (battery,
-                                           // estop_status, speed)
-  ros::Publisher robot_odom_publisher_;    // publish odometry
+  // estop_status, speed)
+  ros::Publisher feedback_publisher_;
+  ros::Publisher robot_odom_publisher_;  // publish odometry
   // parameter variables
   std::string speed_topic_;
   std::string estop_trigger_topic_;
+  std::string mode_trigger_topic_ = "/mode_toggle";
   std::string estop_reset_topic_;
   std::string robot_status_topic_;
-  float robot_status_frequency;
+  float robot_status_frequency_;
   float robot_status_frequency_max = 60;
-  float robot_status_frequency_min = 5;
-  float robot_odom_frequency;
+  float robot_status_frequency_min_ = 5;
```

Comment:
why not use 'const' qualifier for these values. Also it is preferable to use ALL CAPS for defined constants.